src/components/compound/DlCharts/charts/DlDoughnutChart/DlDoughnutChart.vue

```diff
@@ -132,7 +132,9 @@ export default defineComponent({
             default: () => doughnutChartEmptyStateProps
         }
     },
-    setup(props, { attrs }) {
+    setup(props, params) {
+        const attrs = params.attrs
+        const listeners = (params as any).listeners
```

Comment:
Define the `params` with the correct type and remove the `as`, should be something like `{ attrs: ItemClickHandler , listeners: ItemClickHandler }

Reply:
the correct type is this word soup:
<img width="793" alt="Screenshot 2024-02-08 at 1 00 56 PM" src="https://github.com/dataloop-ai/components/assets/146977958/14dc4e1f-d18e-48c4-8459-a02c7a2a6490">
and it does not include `listeners` in Vue2 that's why I did that `as any`, but ok I will think of something better

Reply:
updated PR with that